dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/ejs/container/util/MethodAttribUtils.java

```diff
@@ -369,7 +369,7 @@ public static boolean getConcurrentAsynchronousMethods(Method[] beanMethods) {
                 continue; //onto next method
 
             for (Annotation beanAnno : asynchMethodAnnotations) {
-                if (beanAnno.annotationType().getName().contains("jakarta.enterprise.concurrent.Asynchronous")) {
+                if (beanAnno.annotationType().getName() == "jakarta.enterprise.concurrent.Asynchronous") {
```

Comment:
Good idea using equals to be even more efficient, although `==` will do an instance comparison which will usually work, but there are cases where it might not because there are different instances of the string value, possibly due to different class loaders.  The safer way to do this is:
```
"jakarta.enterprise.concurrent.Asynchronous".equals(beanAnno.annotationType().getName())
```